<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Daniel González</author>
 <class>Chatbot_Window</class>
 <widget class="QMainWindow" name="Chatbot_Window">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Chatbot</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>icon.png</normaloff>icon.png</iconset>
  </property>
  <property name="iconSize">
   <size>
    <width>32</width>
    <height>32</height>
   </size>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>10</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QLabel" name="label">
      <property name="lineWidth">
       <number>1</number>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:10pt; font-weight:600;&quot;&gt;La cultura cubana y sus manifestaciones &lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;a href=&quot;www.grancaribehotels.com/es/home/cultura&quot;&gt;&lt;span style=&quot; font-size:10pt; font-weight:600; text-decoration: underline; color:#0000ff;&quot;&gt;www.grancaribehotels.com/es/home/cultura&lt;/span&gt;&lt;/a&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:10pt; font-weight:600;&quot;&gt;Pregunte lo que desee e intentaré responderle con rapidez y eficiencia&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="textFormat">
       <enum>Qt::AutoText</enum>
      </property>
      <property name="scaledContents">
       <bool>false</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="wordWrap">
       <bool>false</bool>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
      <property name="indent">
       <number>-1</number>
      </property>
      <property name="openExternalLinks">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>13</width>
        <height>25</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPlainTextEdit" name="historial">
      <property name="statusTip">
       <string>Historial de la conversación</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="hLay" native="true">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLineEdit" name="text_input">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>28</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="statusTip">
          <string>Escriba una pregunta</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="send_btn">
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>30</height>
          </size>
         </property>
         <property name="statusTip">
          <string>Realizar pregunta</string>
         </property>
         <property name="text">
          <string>Enviar</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="micro_btn">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
         <property name="statusTip">
          <string>Activar el micrófono para hacer pregunta</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>mic.png</normaloff>mic.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>500</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuArchivo">
    <property name="title">
     <string>Menú</string>
    </property>
    <addaction name="menu_voice_rec"/>
    <addaction name="separator"/>
    <addaction name="menu_cerrar"/>
   </widget>
   <addaction name="menuArchivo"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="menu_opc">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Menu</string>
   </property>
   <property name="statusTip">
    <string>Menu random</string>
   </property>
  </action>
  <action name="menu_cerrar">
   <property name="icon">
    <iconset>
     <normaloff>close.png</normaloff>close.png</iconset>
   </property>
   <property name="text">
    <string>Cerrar</string>
   </property>
   <property name="statusTip">
    <string>Cerrar la aplicación</string>
   </property>
  </action>
  <action name="menu_voice_rec">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>mic.png</normaloff>mic.png</iconset>
   </property>
   <property name="text">
    <string>Activar reconocimiento de voz</string>
   </property>
   <property name="statusTip">
    <string>Seleccione para activar el reconocimiento y síntesis de voz</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>menu_voice_rec</sender>
   <signal>toggled(bool)</signal>
   <receiver>micro_btn</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>475</x>
     <y>555</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
